app/Db/EthercatMC.template

```diff
@@ -0,0 +1,318 @@
+record(motor,"$(PREFIX)$(MOTOR_NAME)")
+{
+	field(DESC,"$(DESC)")
+	field(DTYP,"asynMotor")
+	field(DIR,"$(DIR=0)")
+	field(VELO,"$(VELO=0)")
+	field(JVEL,"$(JVEL=0)")
+	field(VBAS,"$(VBAS=0)")
+	field(ACCL,"$(ACCL=0)")
+	field(JAR, "$(JAR=0)")
+	field(BDST,"$(BDST=0)")
+	field(DLY, "$(DLY=0)")
+	field(BVEL,"$(BVEL=0)")
+	field(BACC,"$(BACC=0)")
+	field(OUT,"@asyn($(MOTOR_PORT),$(AXIS_NO))")
+	field(MRES,"$(MRES=1)")
+	field(RDBD,"$(RDBD=0)")
+	field(ERES,"$(ERES=0)")
+	field(PREC,"$(PREC=3)")
+	field(EGU,"$(EGU=mm)")
+	field(DHLM,"$(DHLM=0)")
+	field(DLLM,"$(DLLM=0)")
+	field(RDBD,"$(RDBD=0)")
+	field(RTRY,"$(RTRY=1)")
+	field(RMOD,"$(RMOD=3)")
+	field(DLY,"$(DLY=0)")
+	field(FOFF,"$(FOFF=0)")
+	field(NTM,"0")
+	field(TWV,"$(TWV=1)")
```

Comment:
Should we slip SPDB in here?

Reply:
Sure - will default it to 0 because of:
https://github.com/epics-modules/motor/blob/855c2540d1b2bd24a9ad2abc72c3f8473d8fa968/motorApp/MotorSrc/motorRecord.cc#L2240